src/transformers/testing_utils.py

```diff
@@ -1476,6 +1477,13 @@ def require_tiktoken(test_case):
     return unittest.skipUnless(is_tiktoken_available(), "test requires TikToken")(test_case)
 
 
+def require_speech(test_case):
+    """
+    Decorator marking a test that requires speech. These tests are skipped when speech isn't available.
+    """
+    return unittest.skipUnless(is_speech_available(), "test requires speech")(test_case)
```

Comment:
Here we have to decide what we really require.

```
def is_speech_available():
    # For now this depends on torchaudio but the exact dependency might evolve in the future.
    return _torchaudio_available
```
and in `setup.py` we have
 
```
extras["speech"] = deps_list("torchaudio") + extras["audio"]
```

If we want to require `torchaudio` + `audio` for the tests, then `is_speech_available` is not enough. 
Also the message `test requires speech` will be confusing as `speech` is not a python library.

If we only want to require `torchaudio`, then we have `def require_torchaudio(test_case):`.

cc @eustlb